package/gluon-mesh-vpn-wireguard/Makefile

```diff
@@ -0,0 +1,20 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=gluon-mesh-vpn-wireguard
+PKG_VERSION:=3
+
+include ../gluon.mk
+
+PKG_CONFIG_DEPENDS += $(GLUON_I18N_CONFIG)
+
+define Package/gluon-mesh-vpn-wireguard
+  TITLE:=WireGuard Mesh VPN Support
+  DEPENDS:=+gluon-core +ip +kmod-wireguard +wireguard-tools +kmod-udptunnel6 +kmod-ipt-hashlimit +micrond
```

Comment:
`kmod-wireguard` already depends on:

```
  DEPENDS:=+IPV6:kmod-udptunnel6 +kmod-udptunnel4
```

Also it's unclear why `kmod-ipt-hashlimit` was included here. We cannot find a use case.

Reply:
Also depend on the `wireguard` package and drop `ip`, `kmod-wireguard` and `wireguard-tools`.

Reply:
And `gluon-mesh-vpn-core` is missing.

Reply:
wireguard package introduces scripts and integration that this package would have to remove. That is why I did not depend on it here. kmod-wireguard and wireguard-tools are dependencies of the wireguard package which is why in absence of +wireguard they are included here. ip is required because the netifd protocol implementation relies on ip. So those aspects are fine.

I did add gluon-mesh-vpn-core and removed the udptunnel/hashlimit kmods.